python-spec/src/somacore/data.py

```diff
@@ -185,7 +190,7 @@ def create(
         uri: str,
         *,
         type: pa.DataType,
-        shape: Sequence[int],
+        shape: Sequence[Union[int, None]],
```

Comment:
sticking the suggestion about `Optional` into its own thread since it’s not exactly related to the rest of the above conversation

Reply:
Thanks the reminder @thetorpedodog -- I'm taking multiple review cycles on 3 PRs simultaneously and have not checked off this particular box just yet but it's on my to-do list for the day. :)

Reply:
Pushed